### PR TITLE
MGMT-16880: assisted-image-service config for ocm 2.10

### PR DESCRIPTION
**Pull request by adriengentil**

### CODE REVIEW

File: ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main__mce.yaml [hunk 1]
```diff
@@ -22,12 +22,12 @@ images:
...
 promotion:
   to:
-  - name: "2.5"
+  - name: "2.6"
     namespace: stolostron
 releases:
   latest:
     integration:
-      name: "4.13"
+      name: "4.16"
```
Comment (danmanor) [inline ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main__mce.yaml:30]: This is unrelated right ?

File: ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.10__mce.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    tag: "8"
+  openshift_release_golang-1.21:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
```
Comment (danmanor) [inline ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.10__mce.yaml:14]: Why is this still 1.19 ?

File: ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.10__mce.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      - registry.ci.openshift.org/openshift/release:golang-1.21
+  to: assisted-image-service
+promotion:
+  to:
+  - name: "2.5"
+    namespace: stolostron
+releases:
+  latest:
+    integration:
+      name: "4.13"
```
Comment (danmanor) [inline ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.10__mce.yaml:30]: Maybe change it to 4.16 as well ?

File: ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main__mce.yaml (hunk 1 above)
Comment (adriengentil) [reply ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main__mce.yaml:30]: yes, unrelated, just to avoid some scripts in dptp to detect this file and do something with it.

File: ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.10__mce.yaml (hunk 2 above)
Comment (adriengentil) [reply ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.10__mce.yaml:14]: why not? :D

Comment (adriengentil) [reply ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.10__mce.yaml:14]: As discussed, we have a lot a things that comes with some history. Let's tackle these issues in a follow-up issue

**Review by osherdp — APPROVED** [review]
